### PR TITLE
Nk pipeline

### PR DESCRIPTION
**Pull request by NadezdaKataeva**
Added test to delete pipeline with test valid name

### CODE REVIEW

File: tests/test_pipeline.py [hunk 1]
```diff
@@ -104,6 +79,8 @@ def test_pipeline_name_in_the_tab(self):
         assert (self.pipelineName_valid in driver.get_element_text(tab_locator_valid))
         assert driver.is_element_present(tab_locator_valid)
 
+
```
Comment (MichaelMicrof) [inline tests/test_pipeline.py:82]: Are you sure that additional lines are necessary?

File: tests/test_pipeline.py [hunk 2]
```diff
@@ -150,3 +127,63 @@ def test_create_project_with_special_symbol_01(self, project_name):
...
+            driver = DashboardPage(self.driver)
+            driver.go_to_page(TD.BASE_URL)
+            driver.get_wait(ProjectLocators.job_by_name(project_name))
+            driver.click(ProjectLocators.job_by_name(project_name))
+            driver.click(ProjectPageLocators.DELETE_PROJECT)
+            driver.get_wait_for_alert()
+            driver.accept_alert()
+            assert driver.is_element_not_present(ProjectLocators.job_by_name(project_name))
+
+
```
Comment (MichaelMicrof) [inline tests/test_pipeline.py:187]: Please, remove empty lines :)

File: tests/test_pipeline.py (hunk 1 above)
Comment (NadezdaKataeva) [reply tests/test_pipeline.py]: Ok, I will remove second assert as not necessarily.

File: tests/test_pipeline.py (hunk 2 above)
Comment (NadezdaKataeva) [reply tests/test_pipeline.py]: done

**Review by MichaelMicrof — APPROVED** [review]
It's not ideal and you can make many changes, but i approved it, because this is at least work.

**Pull request merged**